eks/cdk/fis.py

```diff
@@ -12,10 +13,6 @@ def __init__(self, app: core.App, id: str, props, **kwargs) -> None:
         # copy properties
         self.output_props = props.copy()
 
-        # iam role
-        role = aws_iam.Role(self, "fis-role", assumed_by = aws_iam.ServicePrincipal("fis.amazonaws.com"))
-        role.add_managed_policy(aws_iam.ManagedPolicy.from_aws_managed_policy_name("AdministratorAccess"))
```

Comment:
The FiS role creation code has been moved into the EKS stack to avoid circular dependencies.